README.md

```diff
@@ -12,10 +12,23 @@ git commit
 git push
 ``` 
 
+Or you can make a branch
+
+```sh
+git checkout -b feature/feature-description
+git push --upstream origin feature/feature-description
+# make some changes
+git add .
+git commit -m "implemented feature XYZ"
+git push
+```
+
```

Comment:
Great moves Jerome, keep it up, proud of you!

Reply:
ty fam